presto-main/src/main/java/com/facebook/presto/cost/CostCalculatorUsingExchanges.java

```diff
@@ -169,7 +193,9 @@ public PlanNodeCostEstimate visitExchange(ExchangeNode node, Void context)
         @Override
         public PlanNodeCostEstimate visitTableScan(TableScanNode node, Void context)
         {
-            return cpuCost(getStats(node).getOutputSizeInBytes()); // TODO: add network cost, based on input size in bytes?
+            // memory to cpu througput is about 10GB/s
+            // disk to memory is 100MB/s - 100 slower
+            return cpuCost(getStats(node).getOutputSizeInBytes() * 100); // TODO: add network cost, based on input size in bytes?
```

Comment:
Yes, yes, yes. This is why I wrote that formula is not ideal.

I wanted to be make this a cost higher than the cost of projection.